code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -1570,48 +1586,56 @@ func (srh *StorageRestHandler) getAllocationMinLock(w http.ResponseWriter, r *ht
 		common.Respond(w, r, nil, common.NewErrInternal("no db connection"))
 		return
 	}
-	blobbers, err := getBlobbersForRequest(req, edb, balances, common2.Pagination{})
+
+	var ssc = StorageSmartContract{SmartContract: sci.NewSC(ADDRESS)}
```

Comment:
we should avoid this initialization, we probably should use event's db for it

Reply:
I don't follow. The StorageSmartContract is being used here to add the ADDRESS constant to some MPT keys. We could remove it by replacing `ssc.ID` with `ADDRESS` in several MPT key definitions. Is this worth doing?

The storagesc code could be significantly simplified by a global replacement of ssc.ID or synonyms with ADDRESS.